src/integrations/express.ts

```diff
@@ -83,14 +76,9 @@ function getExpressRouter({ api }: GetExpressRouterContext) {
         return res.json(response)
       } catch (err: any) {
         logger.error('was not able to fetch Navigation: ' + err.message)
-        if (
-          err.message === FSXAApiErrors.NOT_FOUND ||
-          err.message === FSXAApiErrors.UNKNOWN_REMOTE
-        ) {
-          return res.status(404).send(err)
-        } else {
-          return res.status(500).send(err)
-        }
+        res
+          .status(err.statusCode)
+          .send({ message: err.message, error: err.message })
```

Comment:
Message and error are the same? Do we need to keep them both?

Reply:
Yes, this is ugly but it is for backwards compatibility reasons. Up until now "message" has been used in some endpoints to contain the error message. In others there was no error message.
Now, all errors have an error message in "error" and those that used to have it in message still contain it in message. That way old code that relies on a message property doesn't break.

Reply:
I added comments to clarify that